test/integration/test_phone_number_lookup.py

```diff
@@ -135,19 +169,19 @@ def testSuccessfulPhoneNumberLookup(self) -> None:
 
         # Check the information for a Bandwidth TN
         bw_lookup_result: LookupResult = get_lookup_status_response.result[0]
-        self.validateResult(bw_lookup_result, os.environ['BW_NUMBER'], "Bandwidth")
+        self.validateResult(bw_lookup_result, os.environ['BW_NUMBER'], os.environ['BW_NUMBER_PROVIDER'])
 
-        # Check the information for a Verizon TN
-        vzw_lookup_result = get_lookup_status_response.result[1]
-        self.validateResult(vzw_lookup_result, os.environ['VZW_NUMBER'], "Verizon")
+        # # Check the information for a Verizon TN
```

Comment:
FYI: I commented out many tests while I was testing hamcrest `assertions`. Please revert unintentional code merged from that branch-:)